controller: {
    # KP_x: 1.8,
    # KI_x: 0.1,
    # KD_x: 1.5,
    # X_limit: 1.0,
    
    # KP_y: 1.5,
    # KI_y: 0.1,
    # KD_y: 1.2,
    # Y_limit: 1.0,
    
    # KP_z: 3,
    # KI_z: 0.2,
    # KD_z: 3,
    # Z_limit: 1.0,

    KP_x: 1.8,
    KI_x: 0,
    KD_x: 0,
    X_limit: 0.5,
    
    KP_y: 1.5,
    KI_y: 0,
    KD_y: 0,
    Y_limit: 0.5,
    
    KP_z: 3,
    KI_z: 0,
    KD_z: 0,
    Z_limit: 0.5,
    
    KP_yaw: 0.1,
    KI_yaw: 0.05,
    KD_yaw: 0.05,
    
    TAKEOFF_HEIGHT: 0.05,
    TAKEOFF_THROTTLE: 0.35,
    TAKEOFF_THROTTLE_TIME: 1,
    LAND_TIME: 5.0,

    Yaw: 0,
    Flip: 100,
    KP_att: 3.2,
    
    VEHICLE_WEIGHT: 0.25,
}
